app/views/coronavirus_local_restrictions/results.html.erb

```diff
@@ -1,3 +1,9 @@
+<% content_for :back_link do %>
+  <%= render "govuk_publishing_components/components/back_link", {
+    href: find_coronavirus_local_restrictions_path
+  } %>
+<% end %>
```

Comment:
This needs to be added to the "no results" page as well:

<img width="1532" alt="Screenshot 2020-10-26 at 11 15 01" src="https://user-images.githubusercontent.com/5793815/97166127-a49bf580-177c-11eb-9ee6-4c5ebd1187cd.png">

Reply:
Done